libobs/obs-source.c

```diff
@@ -4192,6 +4203,33 @@ void obs_source_output_audio(obs_source_t *source,
 	pthread_mutex_unlock(&source->filter_mutex);
 }
 
+void obs_source_reroute_audio(obs_source_t *dst, obs_source_t *src)
```

Comment:
Just nitpicking, personally I'd make this return a bool indicating success or failure (and then return false on the following `return` calls below), just to be safe if it's needed in the future.

Reply:
Makes sense, it just wasn't useful here. If we can're reroute a source on creation something has gone horribly wrong.